### PR TITLE
Added app from Bnryo

### PR DESCRIPTION
**Pull request by idlip**
+ Record You
+ Trivia You
+ Translate You

They are all modern Material you apps, which are minimal and simple by Bnryo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -426,6 +426,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**PilferShush Jammer**](https://github.com/kaputnikGo/PilferShushJammer) <sup>**[[F-Droid](https://www.f-droid.org/app/cityfreqs.com.pilfershushjammer)]**</sup>
 * [**Privacy Indicator**](https://github.com/NitishGadangi/Privacy-Indicator-App) <sup>**[[F-Droid](https://f-droid.org/app/com.nitish.privacyindicator)]**</sup>
 * [**RustDesk**](https://github.com/rustdesk/rustdesk) <sup>**[[F-Droid](https://f-droid.org/app/com.carriez.flutter_hbb)]**</sup>
+* [**Record You**](https://github.com/Bnyro/RecordYou)
```
Comment (offa) [inline README.md:429]: [No F-Droid](https://github.com/Bnyro/RecordYou/issues/39) unfortunately :-(

File: README.md [hunk 2]
```diff
@@ -540,6 +541,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Brainf**](https://github.com/fredhappyface/android.brainf) <sup>**[[F-Droid](https://f-droid.org/app/com.fredhappyface.brainf)]**</sup>
 * [**ChemistryLab**](https://gitlab.com/asdoi/ChemistryLab)
 * [**phyphox**](https://github.com/phyphox/phyphox-android) <sup>**[[F-Droid](https://f-droid.org/app/de.rwth_aachen.phyphox)]**</sup>
+* [**Trivia You**](https://github.com/Bnyro/TriviaYou) <sup>**[[F-Droid](https://f-droid.org/app/com.bnyro.trivia)]**</sup>
```
Comment (offa) [inline README.md:544]: The source project has been archived and is inactive.

**Review by offa — CHANGES_REQUESTED** [review]
Thanks!

Trivia You is no longer developed accoring to it's Github project, thus isn't suitable here. Otherwise ok :+1:.

File: README.md (hunk 1 above)
Comment (idlip) [reply README.md:429]: Ah, that is why i only added github link.
He says tho, it is as same as in izzydroid.

Are you sure you want me to remove Record you?

I'm fine with it, no issues here.

File: README.md (hunk 2 above)
Comment (idlip) [reply README.md]: Removed!

File: README.md (hunk 1 above)
Comment (offa) [reply README.md:429]: Sorry, no! :see_no_evil: 

The message was intended as info only, so nothing to do here :-).

**Review by offa — APPROVED** [review]

**Comment by offa** [comment]
Thanks again!

**Comment by idlip** [comment]
@offa No worries, Thank you too for taking a deep look ;)

**Pull request merged**